packages/framer-motion/src/animation/types.ts

```diff
@@ -26,6 +26,7 @@ export type ElementOrSelector =
  */
 export interface AnimationPlaybackControls {
     time: number
+    speed: number
```

Comment:
Speed is a bit undescriptive as to what the unit of it is, what about `playbackRate`?

Reply:
That would match WAAPI terminology: https://developer.mozilla.org/en-US/docs/Web/API/Animation/playbackRate

Reply:
IMO `speed`/`scale`/`rate`/`playback-` are all roughly equivalent, `playbackRate` is probably the most descriptive but it is also kind of ugly, and when things are fairly equal I would rather go with APIs that are aesthetically pleasing vs verbose/correct.

```javascript
await animation.finished 🚫
await animation ✅

animation.currentTime = 2 🚫
animation.time = 2 ✅

animation.playbackRate = 0.5 🚫
animation.speed = 0.5 ✅
```

Reply:
Ok, I don't really feel strong about it, so if you're certain about this API, go for it! 👌🏻